### DISCUSSION TITLE
Scheduled Shutdown

### DISCUSSION THREAD

**Discussion by hkhiro** [cockpit-project/cockpit#18910] [Q&A]
so not sure what I am missing (plugin?) but isnt' there a way to 
1. shutdown
2. schedule a shutdown

I do also want to schedule a startup but perhaps need to do that from the bios..

**Comment by hkhiro** [comment]
trying via Services -> Timers tab

**Comment by marusak** [comment]
You can do this, just pick the "Reboot" option from Overview:
![Screenshot from 2023-06-12 17-32-03](https://github.com/cockpit-project/cockpit/assets/12330670/29e6a425-4b46-440b-ab79-de6a5cde25d1)
![Screenshot from 2023-06-12 17-32-18](https://github.com/cockpit-project/cockpit/assets/12330670/757fa212-b53e-401e-9cf7-40f202480528)
